src/frontend/SignatureMismatch.ml

```diff
@@ -365,3 +365,30 @@ let pp_signature_mismatch ppf (name, arg_tys, (sigs, omitted)) =
     name pp_args arg_tys
     (list ~sep:cut pp_signature)
     sigs pp_omitted ()
+
+let pp_math_lib_assignmentoperator_sigs ppf (lt, op) =
+  let signatures =
+    match op with
+    | Operator.Plus | Minus | Times | Divide | EltTimes | EltDivide -> (
```

Comment:
These are all the assignment operators we have, right? I don't see what this match is supposed to do. If it's just guarding against bad inputs, I think it would make more sense to put it inside `make_assigmentoperator_stan_math_signatures`.

Reply:
I think we can actually remove it entirely. It was a holdover from when the function which did this was returning a string option rather than printing